.github/workflows/regression-test-docker-image.yml

```diff
@@ -220,7 +220,7 @@ jobs:
     - name: Check if webperf_core can run Test 18
       if: always()
       run: |
-        testresult=$(docker run ${{ env.TEST_TAG }} python default.py -t 18 -r -i sites.json)
+        testresult=$(docker run --shm-size=4g -e MAX_OLD_SPACE_SIZE=3000 ${{ env.TEST_TAG }} python default.py -t 18 -r -i sites.json)
```

Comment:
@krompaco Test 22 also need this change (as it use lighthouse performance in the background)

Reply:
Ah, thanks. Should be added now.